src/main/java/com/keshavarzi/airplanestats/model/RoleEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.keshavarzi.airplanestats.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "role", schema = "user_data")
+public class RoleEntity {
+    @Id
+    @Nonnull
+    @Column(name = "id", table = "role", nullable = false, unique = true)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long roleId;
+
+    @Nonnull
+    @Column(name = "role_name", table = "role", nullable = false)
+    private String roleName;
```

Comment:
why do you need a role name and an id. Shouldn't the role name be unique already?

Reply:
Want to able to refactor role names (USER to ROLE_USER) later.  What are your thoughts on that?